mender-update/update_module/v3/platform/c++17/fs_operations.cpp

```diff
@@ -370,10 +353,12 @@ error::Error UpdateModule::DeleteStreamsFiles() {
 
 AsyncFifoOpener::AsyncFifoOpener(events::EventLoop &loop) :
 	event_loop_ {loop},
-	cancelled_ {make_shared<atomic<bool>>(true)} {
+	cancelled_ {make_shared<bool>(true)},
```

Comment:
Removing `atomic<>` here. I guess it was never actually needed?

Reply:
Exactly. The intention was to use it in the the thread, but it was never actually used there.